src/components/DeliveryDetails/RecipientForm/RecipientForm.tsx

```diff
@@ -18,7 +18,11 @@ import {
   validateRecipientEmail
 } from './validators';
 
-const RecipientForm = () => {
+type RecipientFormProps = {
+  isMyAccount?: true;
+};
+
+const RecipientForm = ({ isMyAccount }: RecipientFormProps) => {
```

Comment:
`const RecipientForm = ({ isMyAccount = false }: RecipientFormProps) => {`

Reply:
It produced eslint error, but now I found that moving RecipientFormProps to separate file and then importing it fixed it